v0/sources/shopify/util.js

```diff
@@ -66,13 +67,28 @@ const createPropertiesForEcomEvent = message => {
     "root",
     PRODUCT_MAPPING_EXCLUSION_FIELDS
   );
-
   mappedPayload.products = productsList;
   return mappedPayload;
 };
 
+const extractEmailFromPayload = event => {
+  const flattenedPayload = flattenJson(event);
+  let email;
+  const regex_email = new RegExp("\\bemail\\b");
+  const regex_Email = new RegExp("\\bEmail\\b");
```

Comment:
```suggestion
  const email_regex = new RegExp("\\b[eE]mail\\b");
```

For a more generic solution, this should be used:
```suggestion
  const email_regex = new RegExp("\\bemail\\b", "i");
```

Reply:
Got it adding

Reply:
This is addressed